spec/features/cart_spec.rb

```diff
@@ -0,0 +1,40 @@
+# frozen_string_literal: true
+
+feature 'Cart page' do
+  let!(:customer) { create(:customer, :confirmed) }
+  let!(:book) { create(:book, :with_author) }
+  let(:coupon) { create(:coupon, cart: nil) }
+
+  feature 'allows customer to', js: true do
```

Comment:
add few tests for increasing/decreasing books count and removing some book from cart

Reply:
+